packages/vscode-extension/src/OperationResult.ts

```diff
@@ -0,0 +1,16 @@
+export default class OperationResult {
+
+    status: string;
+    resultMessage: string;
+    errorCode: string;
+    errorMessages: Array<string>;
```

Comment:
Change Array<string> for string[] so we declare arrays the same way in all the project